docs/pages/bssw/bssw_content_publishing.md

```diff
@@ -42,10 +42,13 @@ The following steps assume that the BSSw editorial team has already determined t
 * Hyperlinks validity.
 * Check for any prior versions of the same content. If the prior version is very outdated, consider replacing it with this newer content. Please judge on a case-by-case basis.
 * Past events of the same conferences/workshops/etc. are not deleted and just kept as archives.
+* If using [formal citations/references](bssw_styling_originalarticles.html#citationsreferences), use [`wikize_refs.py -i <base>.md`](https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy) to deal with these (see [here](https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/ReferencesInMarkdownHybridApproach.md) for details).
```

Comment:
Please remind me why are "ReferencesInMarkdown*" files sitting in blog directory? Is it because,  historically they were used for blog and put in that folder? Perhaps, we should move them to a different location, outside of content? Is NOW the right time to do it? Perhaps open another PR for this, if needed.